submariner/fine-grained-connectivity.md

```diff
@@ -0,0 +1,87 @@
+# Fine Grained Connectivity
+
+## Summary
+
+Submariner currently provisions full mesh topology by default between the participating clusters.
+For some use-cases, we need fine-grained connectivity options. This proposal covers only the topology
+aspect of connectivity.
+
+This proposal aims to address the below use-case
+
+Hub-Spoke Topology -  Here only the client can talk to the server but not to the other clients to
```

Comment:
Lets not call it hub and spoke, this term means that all nodes can communicate.
https://en.wikipedia.org/wiki/Spoke%E2%80%93hub_distribution_paradigm

Perhaps we can call it just "central cluster" or something and explain the use case.

Reply:
Changed the name.